media-sound/flac123/flac123-0.0.12.recipe

```diff
@@ -0,0 +1,56 @@
+SUMMARY="A command-line program for playing FLAC audio files"
+DESCRIPTION="flac-tools aims to be a complete set of tools for playing and \
+manipulating audio files stored in the FLAC format. flac123 is the only \
+component of flac-tools at this time."
+HOMEPAGE="https://flac-tools.sourceforge.net/"
```

Comment:
lol wait, xd, ok i'll do that i was confused when reading that part of sentence.